src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -362,7 +363,7 @@ internal MergedOptions GetOptions(string authenticationScheme)
 
             try
             {
-                return builder.ExecuteAsync();
+                return builder.ExecuteAsync(tokenAcquisitionOptions != null ? tokenAcquisitionOptions.CancellationToken : CancellationToken.None);
```

Comment:
The standard pattern for cancellation is that every async method ends with the CancellationToken parameter which defaults to None. It's not expected to be in an options object.

I think this would not be a breaking change to this class?

Reply:
This would be a change in the public interface (ITokenAcquisition) ... hence a breaking change.

Reply:
@jmprieur and I synced on this, but to avoid a breaking change, it's going to add a lot more complexity. Basically doubling the API and reducing the developer experience. I think we are leaning towards leaving it as-is and seeing what customers say. When can reconsider at the next major version.